test/scenarios/activegate/installation.go

```diff
@@ -30,12 +29,9 @@ import (
 )
 
 const (
-	agNamespace = "dynatrace"
-	agPodName   = "dynakube-activegate-0"
+	agPodName = "dynakube-activegate-0"
```

Comment:
we remove this and check for the actual dynakube name, since the ag pod is named like the dynakube plus the -activegate-0 suffix